library/types/src/modules/String.rb

```diff
@@ -492,11 +492,11 @@ def ParseOptions(options, parameters)
               # backslah sequences
               backslash_seq = {
                 "a"  => "a", # alert
-                "b"  => "", # backspace
+                "b"  => "\b", # backspace
                 "e"  => "e", # escape
-                "f"  => "", # FF
+                "f"  => "\f", # FF
                 "n"  => "\n", # NL
-                "r"  => " ", # CR
+                "r"  => "\b", # CR
                 "t"  => "\t", # tab
                 "v"  => "v", # vertical tab
```

Comment:
Interestingly, the backslash was lost before `v`. It looks like a different YCP-killer bug.

See the original code before the conversion: https://github.com/yast/yast-yast2/blob/good-bye-ycp/library/types/src/String.ycp#L518